com.trollworks.gcs/src/com/trollworks/gcs/preferences/ReferenceLookupPreferences.java

```diff
@@ -43,14 +55,25 @@
      */
     public ReferenceLookupPreferences(PreferencesWindow owner) {
         super(I18n.Text("Page References"), owner);
-        setLayout(new BorderLayout());
+        setLayout(new PrecisionLayout().setColumns(2));
+        Preferences prefs = Preferences.getInstance();
+
+        //Command line to use to open PDFs
+        String pdfViewerCommandLineTooltip = I18n.Text("The command to launch a PDF viewer to view a page reference. %f for file, %p for page.") +
+                "\n" + I18n.Text("Example") + ":" +
+                "\"C:\\Program Files\\SumatraPDF\\SumatraPDF.exe\" -page %p \"%f\"\n\n" +
```

Comment:
Any example path should be in the native OS's format, i.e. no leading "C:" and forward slashes for all OSs other than Windows.

Reply:
I purposely put a Windows one in because I wasn't sure what a "typical" example would look like elsewhere and was hoping you might have a sample for that.